app/views/listings/new.html.haml

```diff
@@ -0,0 +1,30 @@
+%h1 Create new listing
+
+
+= form_with model: @listing, local: true do |form|
```

Comment:
Are your tests passing with this configuration?

Reply:
Test is passing, but we are also only testing that the form fields are visible and can be filled in at this stage. The actual creation of a listing will be done in a coming PR, and the @listing variable can be declared then. But yes, I guess we should declare it in the `new` section already now.

Reply:
done. declared in the controller.